app/controllers/users_controller.rb

```diff
@@ -53,11 +53,18 @@ class UsersController::InvitationsController < Devise::InvitationsController
 
   include ElevatedPermissionsHelper
 
-  def after_invite_path_for(resource)
-    if resource.admin
+  def after_invite_path_for(current_inviter)
```

Comment:
this param `current_inviter` is not used?

Reply:
No, it's not. But, it's need to be there because the method is overwritten has it. There are to versions:
`after_invite_path_for(current_inviter)`
`after_invite_path_for(current_inviter, resource)`
But there is no one without parameters.
The user is not important for the redirection.